app/src/main/java/nodomain/freeyourgadget/gadgetbridge/model/DeviceType.java

```diff
@@ -60,6 +60,7 @@
     BFH16(140, R.drawable.ic_device_default, R.drawable.ic_device_default_disabled, R.string.devicetype_bfh16),
     MAKIBESHR3(150, R.drawable.ic_device_default, R.drawable.ic_device_hplus_disabled, R.string.devicetype_makibes_hr3),
     MIJIA_LYWSD02(200, R.drawable.ic_device_pebble, R.drawable.ic_device_pebble_disabled, R.string.devicetype_mijia_lywsd02),
+    AMAZFITGTS(210, R.drawable.ic_device_hplus, R.drawable.ic_device_hplus_disabled, R.string.devicetype_amazfit_gts),
```

Comment:
Just make this 19.
I will move Vibratissimo further down. Nobody is using it I guess